src/bootstrap/tool.rs

```diff
@@ -268,6 +268,12 @@ pub fn prepare_tool_cargo(
     if !features.is_empty() {
         cargo.arg("--features").arg(&features.join(", "));
     }
+
+    if builder.config.cmd.bless() {
+        // Bless `expect!` tests.
+        cargo.env("UPDATE_EXPECT", "1");
+    }
```

Comment:
Hm, we might want to just put this in `fn cargo` in Builder?

Reply:
Indeed, that would be more reasonable, updated!